src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1688,6 +1691,73 @@ Expression* TranslateToFuzzReader::makeTry(Type type) {
   return builder.makeTry(body, catchTags, catchBodies);
 }
 
+Expression* TranslateToFuzzReader::makeTryTable(Type type) {
+  if (funcContext->breakableStack.empty()) {
+    // Nothing to break to.
+    // TODO: Perhaps generate a block wrapping us?
+    return makeTrivial(type);
+  }
+  if (wasm.tags.empty()) {
+    addTag();
+  }
+
+  // Add catches of specific tags, and possibly a catch_all at the end. We use
+  // the last iteration of the loop for that.
+  std::vector<Name> catchTags;
+  std::vector<Name> catchDests;
+  std::vector<bool> catchRefs;
+  auto numTags = upTo(MAX_TRY_CATCHES);
```

Comment:
How about `numCatches`, given that some catches don't have tags?

Reply:
Good idea, renamed.